spec/expression.dd

```diff
@@ -1233,9 +1233,8 @@ $(H4 $(LEGACY_LNAME2 CharacterLiteral, character-literal, Character Literals))
 $(H4 String Literals)
 
 $(GRAMMAR
-$(GNAME StringLiterals):
+$(GNAME StringLiteral):
     $(GLINK2 lex, StringLiteral)
-    $(I StringLiterals) $(GLINK2 lex, StringLiteral)
```

Comment:
This isn't right, you are defining `StringLiteral` as being defined as itself. Instead it should link to the `StringLiteral` definition in `lex.dd` in the first hunk.